### PR TITLE
CHE-1298; remove war name from url schema

### PR DESCRIPTION
**Pull request by mshaposhnik**
Consider new direct workspace URL is 
`codenvy.com/namespace/ws_name`

### CODE REVIEW

File: assembly/onpremises-ide-packaging-war-ide/src/main/java/com/codenvy/onpremises/DashboardRedirectionFilter.java [hunk 1]
```diff
@@ -15,12 +15,7 @@
 package com.codenvy.onpremises;
 
 import javax.inject.Singleton;
-import javax.servlet.Filter;
-import javax.servlet.FilterChain;
-import javax.servlet.FilterConfig;
-import javax.servlet.ServletException;
-import javax.servlet.ServletRequest;
-import javax.servlet.ServletResponse;
+import javax.servlet.*;
```
Comment (sleshchenko) [inline assembly/onpremises-ide-packaging-war-ide/src/main/java/com/codenvy/onpremises/DashboardRedirectionFilter.java]: revert these changes

File: assembly/onpremises-ide-packaging-war-ide/src/test/java/com/codenvy/onpremises/DashboardRedirectionFilterTest.java [hunk 2]
```diff
@@ -89,11 +89,29 @@ public void shouldRedirectIfRequestWithoutNamespace(String uri, String url) thro
...
+        return new Object[][]{{"/ws-id/", "http://localhost:8080/ws-id123123/"},
+                              {"/wsname", "http://localhost:8080/wsname_only"},
         };
     }
 
+    @Test
+    public void shouldSkipRequestToTemporaryWs() throws Exception {
+        //given
+        when(request.getMethod()).thenReturn("GET");
+        when(request.getRequestURI()).thenReturn("/ws/ws-id");
```
Comment (sleshchenko) [inline assembly/onpremises-ide-packaging-war-ide/src/test/java/com/codenvy/onpremises/DashboardRedirectionFilterTest.java]: Why do you have "/ws/" here?

File: core/codenvy-hosted-sso-server-organization/src/main/java/com/codenvy/auth/sso/server/OrgServiceUserCreator.java [hunk 3]
```diff
@@ -44,54 +44,54 @@
...
 
-    @Inject
-    private UserDao userDao;
-
-    @Inject
-    private UserProfileDao profileDao;
-
-    @Inject
-    private PreferenceDao preferenceDao;
+    private final UserManager userManager;
```
Comment (sleshchenko) [inline core/codenvy-hosted-sso-server-organization/src/main/java/com/codenvy/auth/sso/server/OrgServiceUserCreator.java]: check formatting here

**Comment by skabashnyuk** [comment]
ok
